tethys_portal/urls.py

```diff
@@ -38,13 +40,23 @@
 # ensure at least staff users logged in before accessing admin login page
 from django.contrib.admin.views.decorators import staff_member_required
 
+prefix_path = ""
+if hasattr(settings, "PREFIX_TO_PATH") and settings.PREFIX_TO_PATH:
+    prefix_path = f"{settings.PREFIX_TO_PATH}"
+
+
 admin.site.login = staff_member_required(
-    admin.site.login, redirect_field_name="", login_url="/accounts/login/"
+    admin.site.login,
+    redirect_field_name="",
+    login_url=f"/{prefix_path}/accounts/login/",
 )
 
+
 admin.autodiscover()
 admin.site.login = staff_member_required(
```

Comment:
I know this was here before, but do you know why admin.site.login is being set twice? Does it still work if one of the definitions is removed?

Reply:
I have taken it away and it seems to work well, and It also pass the tests